ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -9350,7 +9350,16 @@ static int zend_jit_init_fcall(dasm_State **Dst, const zend_op *opline, uint32_t
 			} else if (opline->opcode == ZEND_INIT_FCALL_BY_NAME) {
 				|	LOAD_ADDR FCARG1a, Z_STR_P(zv + 1);
 				|	lea FCARG2a, aword [r2 + opline->result.num]
-				|	EXT_CALL zend_jit_find_func_helper, r0
+				|.if X64
+				|	LOAD_ADDR CARG3, opline
+				|.else
+				|	sub r4, 12
```

Comment:
I'm confused about why it's 12 bytes in other calls I've seen with 3 arguments rather than something smaller for a 4-byte pointer (in the examples I used as reference), but even if it was too large it wouldn't cause bugs